bigquery/storage/managedwriter/options.go

```diff
@@ -234,7 +235,48 @@ func WithTraceID(traceID string) WriterOption {
 // AppendRows calls on the stream.
 func WithSchemaDescriptor(dp *descriptorpb.DescriptorProto) WriterOption {
 	return func(ms *ManagedStream) {
-		ms.curDescVersion = newDescriptorVersion(dp)
+		ms.curTemplate = ms.curTemplate.revise(reviseProtoSchema(dp))
+	}
+}
+
+// WithMissingValueInterpretations controls over how missing values are interpreted
+// for individual columns.
+//
+// You must provide a map to indicate how to interpret missing value for some fields. Missing
+// values are fields present in user schema but missing in rows. The key is
+// the field name. The value is the interpretation of missing values for the
+// field.
+//
+// For example, a map {'foo': NULL_VALUE, 'bar': DEFAULT_VALUE} means all
+// missing values in field foo are interpreted as NULL, all missing values in
+// field bar are interpreted as the default value of field bar in table
+// schema.
+//
+// If a field is not in this map and has missing values, the missing values
+// in this field are interpreted as NULL.
```

Comment:
Maybe mention also: "unless a non-null default was set using `WithDefaultMissingValueInterpretation`" ?

Reply:
updated the docstring here